tests/test_0692_fsspec.py

```diff
@@ -5,11 +5,13 @@
 import uproot
 import uproot.source.fsspec
 
+import skhep_testdata
+
 
 @pytest.mark.network
 def test_open_fsspec_http():
     with uproot.open(
-        "https://github.com/CoffeaTeam/coffea/raw/master/tests/samples/nano_dy.root",
+        skhep_testdata.data_path("uproot-issue121.root"),
         http_handler=uproot.source.fsspec.FSSpecSource,
```

Comment:
Same for this one, and it's not really using the `http_handler`, since it's a local file (as far as Uproot is concerned).

Reply:
Maybe what you want instead is

```
https://github.com/scikit-hep/scikit-hep-testdata/raw/v0.4.33/src/skhep_testdata/data/uproot-issue121.root
```

and then it would be HTTP access for Uproot.